scripts/variantstore/wdl/test/GvsQuickstartHailIntegration.wdl

```diff
@@ -115,8 +115,8 @@ workflow GvsQuickstartHailIntegration {
             git_branch_or_tag = git_branch_or_tag,
             hail_version = effective_hail_version,
             use_VQSR = !use_VETS,
-            avro_path = GvsExtractAvroFilesForHail.avro_prefix,
-            vds_destination_path = GvsExtractAvroFilesForHail.vds_output_path,
+            avro_path = GvsExtractAvroFilesForHail.avro_path,
+            vds_destination_path = GvsExtractAvroFilesForHail.avro_path + "/gvs_export.vds",
```

Comment:
my brain would like a var for this and the corresponding below, but only a nit

Reply:
Done!